keyboards/friedrich/readme.md

```diff
@@ -0,0 +1,19 @@
+# Friedrich
+
+![friedrich](https://i.imgur.com/LH15u37l.jpg)
+
+A small printer friendly 3D printed 50% keyboard.
+
+* Keyboard Maintainer: [kb-elmo](https://github.com/kb-elmo)
+* Hardware Supported: Friedrich PCB by kb-elmo
+* Hardware Availability: [Open source project](https://github.com/MarvFPV/Friedrich)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader for the keyboard, so that it can be flashed?

Reply:
hmmm why exactly?  
haven't seen any instructions like that on other AVR boards so far.

There's absolutely nothing special about this PCB.  
It's either pressing the reset button or using QMK bootmagic.

Reply:
It's a newer requirement, because there are a good number of boards where it is not clear how to enter the bootloader.  Especially if the only way is to use bootmagic. So to make things a bit more user friendly, we'd like to have to have it explicitly stated.

https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> clear instructions on how to reset the board into bootloader mode

Reply:
aight. added :)